doc/api/cli.md

```diff
@@ -426,6 +426,14 @@ unless either the `--pending-deprecation` command line flag, or the
 are used to provide a kind of selective "early warning" mechanism that
 developers may leverage to detect deprecated API usage.
 
+### `--policy-integrity`
```

Comment:
This should be `--policy-integrity=checksum` with some explanation about the expected format of the argument (`checksum` or a more appropriate word if the argument isn't actually a checksum).

Reply:
well it is an SRI string so we could link to that somewhere but the SRI [spec](https://w3c.github.io/webappsec-subresource-integrity/#the-integrity-attribute) is kind of not great for explaining that, would linking to [mdn](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) be fine?

Reply:
Yes, I think linking to mdn should be ok. (and rename my suggested parameter name to `sri` instead of `checksum`).